settings.gradle

```diff
@@ -50,7 +50,8 @@ includeWithFlags ':site'
 
 // Examples
 includeWithFlags ':examples:annotated-http-service', 'java'
-includeWithFlags ':examples:grpc-service',           'java'
+includeWithFlags ':examples:grpc-example',           'java'
```

Comment:
nit: How about `grpc-minimal`? And we could adapt #2310 as `grpc-reactive`?

I'm not sure this suggestion is better than `grpc-example`. 🙃 You can feel free to ignore this.

Reply:
I don't know either. I guess it doesn't matter much anyway though. Let me leave it as it is :wink: